temba/locations/migrations/0003_adminboundary_in_country.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import models, migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('locations', '0002_auto_20141126_2054'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='adminboundary',
+            name='in_country',
+            field=models.CharField(
+                help_text=b"The OSM id of this admin level's country id", max_length=15, null=True),
+            preserve_default=True,
```

Comment:
Sorry for the previous comment about changing migraitons, didn't realize they were just reformatting!

Is this an MPTT thing? I don't like the possibility of having a country_id mismatched with the implied tree through parent.